### ISSUE TITLE
Shorten long URL

### ISSUE THREAD

**Issue by ritazh** [CatalystCode/node-authbot#8]

**Comment by mattdot** [comment]
Is this about putting the whole address in the oauth state?  Facebook will truncate long urls so you'll lose state.

**Comment by ritazh** [comment]
@mattdot Correct. Right now I only have [3 values passed into the state](https://github.com/CatalystCode/node-authbot/blob/master/app.js#L193) but I think as we add more information, we will need to think about how to shorten the URL.
